src/main/java/com/example/ShortnerURL/Controller/PingController.java

```diff
@@ -15,16 +17,18 @@ public class PingController {
     private JdbcTemplate jdbcTemplate;
 
     @GetMapping("/ping")
-    public ResponseEntity<Map<String, String>> ping() {
+    public ResponseEntity<Object> ping() {
         try {
             jdbcTemplate.queryForObject("SELECT current_timestamp;", String.class);
-            Map<String, String> response = new HashMap<>();
-            response.put("status", "ok");
-            return new ResponseEntity<>(response, HttpStatus.OK);
+            SuccessResponseDto successResponseDto = SuccessResponseDto.builder()
+                    .status("status: ok")
+                    .build();
+            return new ResponseEntity<>(successResponseDto, HttpStatus.OK);
         } catch (Exception e) {
-            Map<String, String> response = new HashMap<>();
-            response.put("status", "error");
-            return new ResponseEntity<>(response, HttpStatus.INTERNAL_SERVER_ERROR);
+            ErrorResponseDto errorResponseDto = ErrorResponseDto.builder()
+                    .status("status: error")
```

Comment:
how did you test this?

Reply:
I added an incorrect database source in the application.yaml file, so Spring Boot couldn't connect to the database.